lib/wit/main.py

```diff
@@ -25,6 +25,7 @@
 from .package import WitBug
 from .parser import parser, add_dep_parser
 import re
+from pkg_resources import get_distribution, DistributionNotFound
```

Comment:
I think you don't want to import this stuff in your actual Python program itself, since it causes you to depend on setuptools. Python packaging is kind of a mess and depending directly on setuptools is a pretty good way of running into trouble, so we should try to keep it out of the main app and only in the packaging scripts, if at all.

@robertd-sifive, @nategraff-sifive, do you have any advice here?

Reply:
Mostly just looking for a way to access the version specified from setup.py, happy to try another way

Reply:
Just found https://packaging.python.org/guides/single-sourcing-package-version/

Reply:
Looks like there are a bunch of techniques in https://packaging.python.org/guides/single-sourcing-package-version/. It's been a while since I've done this, but I think there's no one way that is "great", so we just just pick the least evil one.

Reply:
I've removed the __version__ file that was created by the Makefile, created a lib/wit/version.py